packages/sidebar/src/Sidebar.tsx

```diff
@@ -21,116 +23,105 @@ import {
   Separator,
   CloseOpenButton,
   AntiSelect,
-  Scrollbar,
-} from './styled/Sidebar';
+} from './styled';
 
 
-export const PureSidebar = ({
+export const PureSidebar: React.FC<ISidebarProps> = ({
   theme,
   children,
   navComponent,
   showScrollbar,
   arrowComponent = <Arrow />,
+  isResponsive = false,
```

Comment:
I'm not sure, but I think it's possible without this parameter. If children element have fixed width then will be work similar

Reply:
Discussed that isResponsive is unnecessary in our projects so i removed it.